### ISSUE TITLE
Base64 parameter doesn't respect passed value

### ISSUE THREAD

**Issue by Maldaris** [crabbly/Print.js#491]
Given the following call to `printJS`: 
```
      printJS({
        printable: pdf,
        type: 'pdf',
        base64: options.base64 ? true : false
      });
```

PrintJS will always attempt to parse the printable as a base64 string regardless of whether or not the parameter is false.

Cause for this is line 64 in `init.js`:
```
        params.base64 = typeof args.base64 !== 'undefined'
```
Meaning that the parameter is set to whether or not it exists, not the value.

**Comment by crabbly** [comment]
Hey @Maldaris, good catch. The library never accounted for `base64` being passed as false, we pretty much assumed that it would just not get passed in at all. I'll get a commit up to fix this. Thanks for reporting it 🚀

**Comment by crabbly** [comment]
@Maldaris Issue was fixed here:
https://github.com/crabbly/Print.js/commit/37eaeaf77d7103aa5d8a5fce9dc5cf78db51a495

Thanks again for pointing this out. I'll post an update here once this change has been published to npm.

**Comment by crabbly** [comment]
This change was published to npm with v1.5.0. Please run `npm update print-js` to get it. Thank you

**Comment by Maldaris** [comment]
> we pretty much assumed that it would just not get passed in at all.

In our case we might have a data URL or an actual network resource, so determining which was necessary up dynamically. Thanks for your quick response!
